packages/models/project.json

```diff
@@ -5,6 +5,7 @@
   "projectType": "library",
   "targets": {
     "build": {
+      "dependsOn": ["types-generation"],
```

Comment:
We don't need to generate types in order to build (also the target name is now `generate-types`). 

```suggestion
```